.gitleaks.toml

```diff
@@ -0,0 +1,2 @@
+[exclude]
+  paths = ["refs/remotes/*"]
```

Comment:
This has generally been our existing approach, but the drawback is that we have to specify this config file per repository. The alternatives would be a giant base MegaLinter config or config inheritance per linter for those that support it. Any thoughts?

Reply:
I see what you're saying about this being a drawback. The giant base MegaLinter config idea sounds unappealing/hard to parse through and understand cognitively. I'm not sure what config inheritance per linter would look like exactly, but that sounds tentatively pretty appealing.

Reply:
See the extend section of https://github.com/gitleaks/gitleaks#configuration.

Reply:
Cool--I like that idea. I'll change the commit to a "feat" so there will be a new release and we can extend from the change in other repos.

Reply:
After experimentation, it seems like using the Gitleaks configuration file did not accomplish the intended goal. Adding a `--log-opts` argument to the MegaLinter config did. Sorry it seems that we are bloating the base MegaLinter config but I'm not aware of a better way at the moment. If you have any thoughts on how to accomplish this in a better way, let me know!

Reply:
That's totally fine. Let's test whether this triggers any false positives in any of our repositories before merging this though.

Reply:
Testing complete!